fence/agents/rhevm/fence_rhevm.py

```diff
@@ -136,9 +146,26 @@ def define_new_opts():
 		"required" : "0",
 		"shortdesc" : "Reuse cookies for authentication",
 		"order" : 1}
+	all_opt["api_path"] = {
+		"getopt" : "",
+		"longopt" : "api-path",
+		"help" : "--api-path                     The path part of the API URL",
```

Comment:
@oalbrigt I added that line, but it doesn't have any effect on the generated XML. Is that OK?